web.config

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this is imminently moving to AKS so I recommend reverting this

Reply:
is the above still valid? if so, why?

Reply:
yes still valid, this was moved to AKS months ago

Reply:
file removed.